lachsheshanit.py

```diff
@@ -0,0 +1,24 @@
+def main():
```

Comment:
Please make separation between main and functions - you should write it in functions and call them in main.
Remember to add documantation to the functions you are about to write.